### PR TITLE
[MOSIP-24203] Added documentation to back up & restore k8s cluster via ETCD

### PR DESCRIPTION
**Pull request by syedsalman3753**

### CODE REVIEW

File: docs/etcd-backup-restore-via-rke.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  ....
+  ....
+  INFO[0265] [addons] Setting up user addons              
+  INFO[0265] [addons] no user addons defined              
+  INFO[0265] Finished building Kubernetes cluster successfully
+  INFO[0265] Restarting network, ingress, and metrics pods
+  I0201 21:09:42.691615   14481 request.go:665] Waited for 1.043236434s due to client-side throttling, not priority and fairness, request: DELETE:https://172.31.6.129:6443/api/v1/namespaces/kube-system/pods/coredns-8578b6dbdd-776g5?timeout=30s
+  INFO[0268] Finished restoring snapshot [soil-2024-02-01T20:23:31Z_etcd.zip] on all etcd host
+  ```
+  * Once the restore operation successfully executed, Check the status of nodes, & pods of k8s cluster.
```
Comment (SwatiPeriwal) [inline docs/etcd-backup-restore-via-rke.md:95]: Once the restore operation is successfully executed, check the status of nodes and pods of k8s cluster.

File: docs/etcd-backup-restore-via-rke.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      ....
+      ....
+      backup_config:
+        interval_hours: 12
+        retention: 6
+  ```
+
+## Restore
+
+* To restore the state of the Kubernetes (k8s) cluster from a ETCD snapshot, execute the following command after updating the name of the snapshot (i.e., `<ETCD-SNAPSHOT-NAME>`).
```
Comment (SwatiPeriwal) [inline docs/etcd-backup-restore-via-rke.md:72]: Why is ETCD capitalised in some places while it is etcd is the remaining? pls verify and update accordingly.

**Review by SwatiPeriwal — CHANGES_REQUESTED** [review]
Pls review and update as needed..

**Review by SwatiPeriwal — APPROVED** [review]
Approving changes

**Review by ckm007 — APPROVED** [review]

**Pull request merged**